src/lib/openapi/spec/playground-strategy-schema.ts

```diff
@@ -51,6 +53,54 @@ export const strategyEvaluationResults = {
                     description:
                         'Whether this strategy evaluates to true or not.',
                 },
+                variant: {
```

Comment:
Created inline variant schema that reflects what SDK sees. It's different from the variant in the admin API which has more fields.